scripts/build_torch_wheels.sh

```diff
@@ -213,9 +213,14 @@ function build_armcomputelibrary() {
 
 function build_and_install_torch() {
   # Checkout the PT commit ID or branch if we have one.
-  COMMITID_FILE="xla/.torch_pin"
-  if [ -e "$COMMITID_FILE" ]; then
-    git checkout $(cat "$COMMITID_FILE")
+  TORCH_PIN="xla/.torch_pin"
+  if [ -e "$TORCH_PIN" ]; then
+    git checkout $(cat "$TORCH_PIN")
```

Comment:
How does this work? I thought this file contains a PR number, and I don't think `git checkout PR_NUMBER` works.